codegens/php-pecl-http/.editorconfig

```diff
@@ -0,0 +1,14 @@
+# editorconfig.org
+root = true
```

Comment:
can we move .editorconfig to the root repo?
Also remove it from other folders if it exists